packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -136,16 +145,13 @@
     </dd>
     <dt>valign <code>enum</code></dt>
     <dd>
-      <p>If set, determines the vertical alignment of table's cell (td) content. While the acceptable values contain all
-        of the values that the CSS property accepts, the default (top) and middle are the values most likely to be used.</p>
+      <p>If set, determines the vertical alignment of table's cell (td) content. While it is set to top by default,
+        content will be middle aligned vertically unless there are two or more lines of contentin any cell in a given
```

Comment:
```suggestion
        content will be middle aligned vertically unless there are two or more lines of content in any cell in a given
```

Reply:
I am not sure this statement is clear, for a reader. 
- if by default the alignment is top, how can be _middle aligned vertically_?
- this may be true for text-only content, but we have to consider every possible content (eg. an image/logo or a custom content/component)

I suggest to remove entirely the explanation (`While it is ... in a give row`), to avoid confusion.